static/js/activity.js

```diff
@@ -127,8 +127,14 @@ exports.get_huddles = function () {
     return huddles.reverse();
 };
 
+function huddle_split(huddle) {
+    return _.map(huddle.split(','), function (s) {
+        return parseInt(s, 10);
+    });
+}
```

Comment:
When written as `huddle.split(',').map(s => parseInt(s, 10))`, this doesn’t seem worth extracting as a helper function.

Reply:
Keeping this as is.  Makes it easy to look at the two callers and see what code is in common.

Reply:
I actually changed it as a separate commit (not pushed yet).